spyder/plugins/completion/providers/kite/widgets/calltoaction.py

```diff
@@ -91,7 +88,7 @@ def handle_mouse_press(self, event):
         self.hide()
 
     def handle_processed_completions(self, completions):
-        if not self._enabled:
+        if not CONF.get('completions', 'kite_call_to_action'):
```

Comment:
Please make this class inherit from `SpyderConfigurationAccessor` and use `self.get_conf/self.set_conf` instead. That's the new way to access config options instead of calling `CONF` directly.

Reply:
You also need to set `CONF_SECTION = 'completions'` as a class constant for that to work.